backend/authentication/routes.py

```diff
@@ -37,6 +52,40 @@ def get(self, token):
                }, 200
 
 
+# Callback route to get data from auth0
+class UserCallBack(Resource):
+    def get(self):
+        auth0.authorize_access_token()
+        resp = auth0.get('userinfo')
+        userinfo = resp.json()
+
+
+        session['jwt_payload'] = userinfo
+        session['profile'] = {
+            'name': userinfo['name'],
+            'image': userinfo['picture'],
+            'username': userinfo['email']
+        }
+
+        # Hard coding this data for now
+        userinfo["track_id"] = 1
+        userinfo["location"] = "Davis"
+        # I think is_active is now unneccessary
+        userinfo["is_active"] = True 
+        userinfo["password"] = "TemporaryPW"
+
+        # If user doesn't exist store in db
+        cur_user = db.session.query(User).filter_by(username=userinfo["email"]).first()
```

Comment:
You can user User.query.filter_by(username=userinfo["email"]).first() for a shorter query